plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/MockAmazonS3.java

```diff
@@ -54,6 +55,17 @@
     // length of the input data is 100 bytes
     private byte[] byteCounter = new byte[100];
 
+
+    private void openSocket() {
+        try {
+            Socket socket = new Socket("localhost", 9200);
```

Comment:
Are there any guidelines which ports could be used in unit/integration tests? I think I wasn't able to use any other port than 9200 without changing further permissions.